docs/dyndns/configure.rst

```diff
@@ -36,11 +36,8 @@ following details in your router configuration:
 - Hostname (same as your username)
 - Token secret (long random string for authentication, displayed after sign-up)
 
-**Advanced API users:** The dynDNS token technically is a regular API token.
-You can also use the token to make requests to our REST API. (Currently, all
-tokens are equally powerful, i.e. a token used for dynDNS updates can also be
-used to perform other kinds of API operations. Token scoping is on our
-roadmap.)
+**Advanced API users:** The dynDNS token technically is a regular API token
+with permissions restricted to DNS management (but not account management).
```

Comment:
These are the tokens that are generated for immediate display when activating an account that includes a dynDNS registration, right?

In that case, I don't think this is true. Or do you mean something else?

Reply:
When an account is activated using `AuthenticatedActivateUserActionView` (`api/v1/v/activate-account/`) and a locally registrable domain is given, a token object is returned in the response. [This token is created using `Token.objects.create(user=domain.owner, name="dyndns")`](https://github.com/desec-io/desec-stack/blob/830e9ba5724ecb6f0137b33118c3d1101c102876/api/desecapi/views/authenticated_actions.py#L167), i.e. the default value of `False` for `perm_manage_tokens` applies.

This makes sense, as we expect this token to end up in some router's storage; why should it be able to manage the deSEC account?

What is a bit awkward is that this business is done by the view class, rather than the action itself.

Reply:
> i.e. the default value of `False` for `perm_manage_tokens` applies.

Right! I thought the default was `True`. My bad!